pkg/domain/vulnerability_test.go

```diff
@@ -0,0 +1,81 @@
+package domain_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kptm-tools/common/common/pkg/enums"
+	"github.com/kptm-tools/core-service/pkg/domain"
+)
+
+func TestMapZapRiskToSeverity(t *testing.T) {
+	tests := []struct {
+		name       string
+		risk       enums.RiskCodeType
+		confidence enums.ConfidenceWebScanType
+		expected   enums.SeverityType
+	}{
+		{
+			name:       "High Risk with High Confidence",
+			risk:       enums.RiskCodeHigh,
+			confidence: enums.ConfidenceHigh,
+			expected:   enums.SeverityTypeCritical,
+		},
+		{
+			name:       "High Risk with False Positive Confidence",
+			risk:       enums.RiskCodeHigh,
+			confidence: enums.ConfidenceFalsePositive,
+			expected:   enums.SeverityTypeMedium,
+		},
+		{
+			name:       "High Risk with Default Confidence",
+			risk:       enums.RiskCodeHigh,
+			confidence: enums.ConfidenceMedium,
+			expected:   enums.SeverityTypeHigh,
+		},
+		{
+			name:       "Medium Risk with High Confidence",
+			risk:       enums.RiskCodeMedium,
+			confidence: enums.ConfidenceHigh,
+			expected:   enums.SeverityTypeMedium,
+		},
+		{
+			name:       "Medium Risk with Default Confidence",
+			risk:       enums.RiskCodeMedium,
+			confidence: enums.ConfidenceMedium,
+			expected:   enums.SeverityTypeLow,
+		},
+		{
+			name:       "Low Risk with False Positive Confidence",
+			risk:       enums.RiskCodeLow,
+			confidence: enums.ConfidenceFalsePositive,
+			expected:   enums.SeverityTypeNone,
+		},
+		{
+			name:       "Low Risk with Default Confidence",
+			risk:       enums.RiskCodeLow,
+			confidence: enums.ConfidenceMedium,
+			expected:   enums.SeverityTypeLow,
+		},
+		{
+			name:       "Informational Risk",
+			risk:       enums.RiskCodeInformational,
+			confidence: enums.ConfidenceHigh,
+			expected:   enums.SeverityTypeNone,
+		},
+		{
+			name:       "Unknown Risk",
+			risk:       "",
+			confidence: enums.ConfidenceHigh,
+			expected:   enums.SeverityTypeUnknown,
+		},
```

Comment:
`[Suggestion]` Although it's the same behavior, we might be missing the test where confidence is an empty string or "Unknown":

 
```suggestion
		{
			name:       "Unknown Risk",
			risk:       "",
			confidence: enums.ConfidenceHigh,
			expected:   enums.SeverityTypeUnknown,
		},
				{
			name:       "Unknown Confidence",
			risk:       enums.RiskCodeInformational,
			confidence: "",
			expected:   enums.SeverityTypeUnknown,
		},
```